optimum/exporters/openvino/model_configs.py

```diff
@@ -1488,6 +1488,7 @@ def __init__(
         float_dtype: str = "fp32",
         behavior: LlavaConfigBehavior = LlavaConfigBehavior.VISION_EMBEDDINGS,
         preprocessors: Optional[List[Any]] = None,
+        **kwargs,
```

Comment:
why is it needed for MAIRA-2 models ?

Reply:
maira processor has own arguments different from standard, this is needed to support inference for this case

Reply:
automatic task detection in some cases select text-generation-with-past as task this is required to pass some extra arguments to config